src/client/actions/GuildBanRemove.js

```diff
@@ -14,7 +14,11 @@ class GuildBanRemove extends Action {
      * @param {Guild} guild The guild that the unban occurred in
      * @param {User} user The user that was unbanned
      */
-    if (guild && user) client.emit(Events.GUILD_BAN_REMOVE, guild, user);
+    if (guild && user) {
+      const ban = guild.bans.cache.get(user.id);
```

Comment:
if a ban is uncached here then the event will emit `undefined`, so there should be something done to fix this

Reply:
b76e2b5